### PR TITLE
Provide a refresh task for syncing state without an apply

### PR DESCRIPTION
**Pull request by ody**
Sometimes you want to resync state instead of doing an apply to capture modifications to resources which occurred outside of previous Terraform plan run.

...also fixes tests which were found broken

### CODE REVIEW

File: tasks/refresh.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require_relative '../lib/cli_helper.rb'
+
+class TerraformOutput < TaskHelper
```
Comment (nicklewis) [inline tasks/refresh.rb:7]: The references to `output` throughout this file should be `refresh`

**Comment by ody** [comment]
Tests failed because change isn't being picked up which should be present...do not know why

File: tasks/refresh.rb (hunk 1 above)
Comment (ody) [reply tasks/refresh.rb]: Thanks for the catch.

**Comment by ody** [comment]
@nicklewis Do you know if testing this module under Bolt 1.x is still required? I updated tests to be successful in my development environment, which install Bolt 3.x.

**Comment by nicklewis** [comment]
No, Bolt 1.x hasn't been supported since we released 2.0.

**Review by donoghuc — APPROVED** [review]
👍 thanks!

**Pull request merged**